Document the minimum required Node.js engine to execute nexe (v2.x)

This is a
 - [x] Feature Request

Could we please document the minimum required Node.js engine to **execute** nexe (v2.x)?

I was able to build (`npm run build`) it using `node@4.8.3` but I am not able to run it as the resulting built `nexe` still contains [at least] some more modern syntax like the ES6 spread operator, which means it requires `node@>=5.0.0`, if not higher.

Given the presence of Babel in the build, I assumed it would back-compile everything to ES5 syntax but apparently that is not the case -- which is fine, I'm just unsure of the hard requirements.

e.g.

```
C:\git\Observer\Node\ObsNode\node_modules\nexe\lib\nexe.js:22
    const nexe = (0, _appBuilder.compose)(...[_resource2.default, _bundle2.default, _cli2.default, build && _download2.default, build && _artifacts2.default, build && _patches2.default, build && options.patches].filter(function (x) {
                                          ^^^

SyntaxError: Unexpected token ...
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (C:\git\Observer\Node\ObsNode\node_modules\nexe\index.js:2:14)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
```

--- a/package.json
+++ b/package.json
@@ -26,6 +26,9 @@
   "bin": {
     "nexe": "index.js"
   },
+  "engines": {
+    "node": ">=6"
+  },
   "dependencies": {
     "app-builder": "^5.1.0",
     "bluebird": "^3.5.0",

